core/io/resource_saver.h

```diff
@@ -82,9 +82,12 @@ class ResourceSaver {
 		FLAG_SAVE_BIG_ENDIAN = 16,
 		FLAG_COMPRESS = 32,
 		FLAG_REPLACE_SUBRESOURCE_PATHS = 64,
+		FLAG_SAVE_INTEGRITY_NONE = 128,
+		FLAG_SAVE_INTEGRITY_SAVE_SWAP = 256,
+		FLAG_SAVE_INTEGRITY_SAVE_SWAP_PLUS_SYNC = FLAG_SAVE_INTEGRITY_NONE | FLAG_SAVE_INTEGRITY_SAVE_SWAP,
```

Comment:
Are these flags used anywhere?

Reply:
Nooo!!!!

This is a left-over from a previous, scrapped idea. Well spotted! Removing this as well.